sentinelsat/sentinel.py

```diff
@@ -1183,41 +1183,44 @@ def name_from_path(path):
 
             # Collect
             if name not in names_from_paths:
-                paths.append(join(directory, name + ".zip"))
+                paths.append(directory / (name + ".zip"))
```

Comment:
Not sure, should we use ``_get_filename`` here?

Reply:
Good catch. Thanks! Fixed it.